src/Nethermind/Nethermind.Network/P2P/Subprotocols/Eth/V65/Eth65ProtocolHandler.cs

```diff
@@ -52,11 +54,20 @@ public override void HandleMessage(ZeroPacket message)
             switch (message.PacketType)
             {
                 case Eth65MessageCode.PooledTransactions:
-                    PooledTransactionsMessage pooledTxMsg
-                        = Deserialize<PooledTransactionsMessage>(message.Content);
-                    Metrics.Eth65PooledTransactionsReceived++;
-                    ReportIn(pooledTxMsg, size);
-                    Handle(pooledTxMsg);
+                    if (CanReceiveTransactions)
```

Comment:
NP: Could've skipped in `Eth62ProtocolHandler.Handle(TransactionsMessage)`. Reduce amount of change.

Reply:
Yes but I wanted to avoid deserialization too.